src/routes/recipe.js

```diff
@@ -1,17 +1,62 @@
 import React from 'react';
 import { useParams } from "react-router-dom";
 import { Link } from "react-router-dom";
+import { getRecipe } from '../recipes';
+
+function IngredientItem(props) {
```

Comment:
Minor: It's pretty common in React to spread the props out up here in the component definition. This makes it simpler to access the props in the component (no need for `prop.`) and also serves as quick documentation at the top of which props your component takes. I'd additionally suggest using either TypeScript or [React Prop-Types](https://reactjs.org/docs/typechecking-with-proptypes.html) to document props, but that might understandably be more than you want to take on at this point!

```suggestion
function IngredientItem({ ingredients, instructions, keyIngredients }) {
```